tests/conftest.py

```diff
@@ -46,7 +57,8 @@ def app_config(app_config):
     https://github.com/inveniosoftware/invenio-records-permissions/issues/51
     """
     app_config["RECORDS_REST_ENDPOINTS"] = {}
-
+    app_config["INDEXER_DEFAULT_DOC_TYPE"] = "testrecord"
```

Comment:
normal/enhancement: we can set up also [`SEARCH_MAPPINGS`](https://invenio-search.readthedocs.io/en/latest/configuration.html#invenio_search.config.SEARCH_MAPPINGS) variable. That way, only this mapping will be created and not also the others specified in other modules entrypoints (Im quite sure the records-... is being created anyway).

Reply:
I was thinking once we sever ties with invenio-records-rest/invenio-files-rest, we won't have other modules creating mappings that this module depends on. This should resolve itself with https://github.com/inveniosoftware/invenio-records-permissions/issues/51 and once we do files next month.